homeassistant/components/openhome/media_player.py

```diff
@@ -123,6 +126,20 @@ def turn_off(self):
         """Put device in standby."""
         self._device.SetStandby(True)
 
+    def play_media(self, media_type, media_id, **kwargs):
+        """Send the play_media command to the media player."""
+        if not media_type == MEDIA_TYPE_MUSIC:
+            _LOGGER.error(
+                "Invalid media type %s. Only %s is supported",
+                media_type,
+                MEDIA_TYPE_MUSIC,
+            )
+            return
+        track_details = {}
+        track_details["title"] = "Home Assistant"
+        track_details["uri"] = media_id
```

Comment:
Just one last suggestion:
```python
track_details = {
    "title": "Home Assistant"
    "uri": media_id
}
```

Reply:
Good suggestion - I've implemented that now and pushed the change :)